example/spec/ingest/ingestFromPdrSpec.js

```diff
@@ -0,0 +1,441 @@
+'use strict';
+
+/**
+ * End to end ingest from discovering a PDR
+ *
+ * Kick off discover and queue pdrs which:
+ * Discovers 1 PDR
+ * Queues that PDR
+ * Kicks off the ParsePDR workflow
+ *
+ * Parse PDR workflow:
+ * parses pdr
+ * queues a granule
+ * pdr status check
+ * This will kick off the ingest workflow
+ *
+ * Ingest worklow:
+ * runs sync granule - saves file to file staging location
+ * performs the fake processing step - generates CMR metadata
+ * Moves the file to the final location
+ * Does not post to CMR (that is in a separate test)
+ */
+
+const { Collection, Execution, Pdr } = require('@cumulus/api/models');
+const {
+  buildAndExecuteWorkflow,
```

Comment:
Super-picky, but it's easier to find an import if these are sorted alphabetically.